.github/workflows/changelog-v3.yml

```
@@ -1,25 +1,238 @@
+          echo "v3/scripts/validate-changelog.go" >> .git/info/exclude
+          # Get the correct branch name to push to
+          REPO_OWNER="wailsapp"  # Always wailsapp for this repo
+          
+          if [ "${{ github.event_name }}" = "pull_request" ]; then
+            BRANCH_NAME="${{ github.event.pull_request.head.ref }}"
+          else
+            # For manual workflow dispatch, get PR info
+            PR_INFO=$(gh pr view ${{ steps.pr_info.outputs.pr_number }} --json headRefName,headRepositoryOwner)
+            BRANCH_NAME=$(echo "$PR_INFO" | jq -r '.headRefName')
+            HEAD_REPO_OWNER=$(echo "$PR_INFO" | jq -r '.headRepositoryOwner.login')
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)